src/user/mod.rs

```diff
@@ -419,54 +391,3 @@ pub fn updates(req: &mut Request) -> CargoResult<Response> {
         meta: Meta { more: more },
     }))
 }
-
-#[cfg(test)]
```

Comment:
Nope, I can readd and update them. They just seemed very low value tests with the tokens pulled out to be a separate resource.